datalad/distribution/add.py

```diff
@@ -119,8 +119,16 @@ class Add(Interface):
             transport"""),
         recursive=recursion_flag,
         recursion_limit=recursion_limit,
+        ds2super=Parameter(
+            args=("--ds2super", "--datasets-to-super",),
```

Comment:
But this command takes files and directories that are not necessarily datasets. Without knowing the details one could get the impression that one can add a file to a superdataset. Wanted to avoid that.